libqpdf/QPDF.cc

```diff
@@ -1292,48 +1292,48 @@ QPDF::showXRefTable()
     }
 }
 
-// Ensure all objects in the pdf file, including those in indirect references,
-// appear in the object cache.
 void
 QPDF::fixDanglingReferences(bool force)
 {
+    // Ensure all objects in the pdf file, including those in indirect
+    // references, appear in the object cache.
```

Comment:
With the comment above the void, visual editors will display the comment (together with the method signature) as a tool tip wherever the method is called.

With the comment inside the method body, the comment will not be displayed.

Reply:
Okay, I didn't know why you had deviated from my convention of putting the comment inside the function. I keep the comment inside the function because emacs handles that case better.

Reply:
I'm going to explore lsp-mode in emacs and see if I can get some more modern IDE features. If there are certain conventions that we can follow uniformly that help out with various IDEs or visual editors, I'm open to it. This may be another one of those cases where we do a bulk change, but we should agree on it first. I want to make sure I don't break my emacs setup though or that I can rework it so that it handles changes properly. That said, I have some tolerance for issues here. I've gotten my emacs configuration for automatic indentation to work _almost_ like clang-format so that, most of the time, my code matches what the ultimate format will be. Then I just do a clang-format-buffer to fix it and don't mess around with it from inside emacs. The most destructive emacs operation is to indent a block, but I never do that anymore with clang-format-buffer. Anyway, you've mentioned the comment outside the function here, and I know you've mentioned other things other places but I don't recall the details. I'll open a discussion ticket.